Dockerfile

```diff
@@ -186,12 +149,23 @@ RUN make entrypoints
 RUN --mount=type=cache,target=/root/.cache \
     --mount=type=cache,target=/var/lib/localstack/cache \
     source .venv/bin/activate && \
+    python -m localstack.cli.lpm install java --version 11 && \
     python -m localstack.cli.lpm install \
       lambda-runtime \
       dynamodb-local && \
     chown -R localstack:localstack /usr/lib/localstack && \
     chmod -R 777 /usr/lib/localstack
 
+# Set up Java
+ENV JAVA_HOME /usr/lib/localstack/java/11
+RUN { \
+        echo '#!/bin/sh'; echo 'set -e'; echo; \
+        echo 'dirname "$(dirname "$(readlink -f "$(which javac || which java)")")"'; \
+    } > /usr/local/bin/docker-java-home \
+    && chmod +x /usr/local/bin/docker-java-home
```

Comment:
Is this code here still necessary? I can't really see where `docker-java-home` is being used...

Reply:
I wasn't sure whether or not to remove it as it could be used in non-LS or bigdata images.

For now, removed it in c108cb5c2, we could revert if required